### DISCUSSION TITLE
activate pointer from outside the chart

### DISCUSSION THREAD

**Discussion by AkramSyed002** [Abhinandan-Kushwaha/react-native-gifted-charts#383] [Q&A] [answered]
i need to show the pointer on the chart from out outside label once click on the i tried but did found any solution 
is there any way to do like this SS attached

<img width="288" alt="Screenshot 2023-09-09 at 4 55 06 PM" src="https://github.com/Abhinandan-Kushwaha/react-native-gifted-charts/assets/68083755/ecc2231b-7199-4bab-8c72-95e9c0c8e1a1">

**Comment by Abhinandan-Kushwaha** [comment, marked as answer]
Hi @AkramSyed002 👋
Thanks for bringing this out. I don't think this is currently supported, but I can suggest you a workaround.

This can be achieved using the `dataPointRadius` and `showVerticalLine` properties inside data items, and using `onPressIn` and `onPressOut` with your external touchable component.

Here is an example-

![pointersFromOutside](https://github.com/Abhinandan-Kushwaha/react-native-gifted-charts/assets/20596944/6ef01fba-c563-464e-9ac9-ed0395a55a23)

The code for the above chart is-
```js
const App = () => {
  const [lineData, setLineData] = useState([
    {value: 0},
    {value: 10},
    {value: 8},
    {value: 58},
    {value: 56},
    {value: 78},
    {value: 74},
    {value: 98},
  ]);

  const days = ['Mon','Tue','Wed','Thu','Fri','Sat','Sun','Mon']

  const showOrHidePointer = (ind, show) => {
    setLineData(
      lineData.map((item, index) => {
        if (index === ind) {
          item.dataPointRadius = show ? 8 : 0;
          item.showVerticalLine = show
        }
        return item;
      }),
    );
  };

  return (
    <View>
      <View style={{flexDirection: 'row', marginLeft: 8}}>
        {lineData.map((item, index) => {
          return (
            <TouchableOpacity
              key={index}
              style={{
                padding: 6,
                margin: 4,
                backgroundColor: '#ebb',
                borderRadius: 8,
              }}
              onPressIn={() => showOrHidePointer(index, true)}
              onPressOut={() => showOrHidePointer(index, false)}>
              <Text>{days[index]}</Text>
            </TouchableOpacity>
          );
        })}
      </View>
        <LineChart
          data={lineData}
          curved
          dataPointsRadius={0}
          dataPointsColor='#f88'
          verticalLinesColor={'#f88'}
          initialSpacing={0}
          spacing={45}
        />
    </View>
  );
}
```

**Comment by Abhinandan-Kushwaha** [comment]
@AkramSyed002  Please consider using the latest version of the library i.e. `1.3.6`
It has some minor fixes you might need.
